program/src/state.rs

```diff
@@ -76,6 +222,11 @@ pub(crate) fn collect_holder_rewards_pool_signer_seeds<'a>(
 #[derive(Clone, Copy, Debug, Default, PartialEq, Pod, ShankAccount, Zeroable)]
 #[repr(C)]
 pub struct HolderRewards {
+    /// The rewards per token exchange rate when this holder last harvested.
+    ///
+    /// Stored as a `u128`, which includes a scaling factor of `1e9` to
+    /// represent the exchange rate with 9 decimal places of precision.
+    pub last_rewards_per_token: u128,
     /// The last seen total rewards amount in the aggregate holder rewards
     /// account.
     pub last_seen_total_rewards: u64,
```

Comment:
With `last_rewards_per_token`, we don't need this field anymore!

Reply:
Ha, I thought so! I have the changes locally to strip this, and `total_rewards`. I'll push it up.